server/src/main/java/org/opensearch/index/engine/Engine.java

```diff
@@ -1321,6 +1321,7 @@ public void failEngine(String reason, @Nullable Exception failure) {
                     // clean all unreferenced files on best effort basis created during failed merge and reset the
                     // shard state back to last Lucene Commit.
                     if (shouldCleanupUnreferencedFiles() && isMergeFailureDueToIOException(failure, reason)) {
+                        logger.info("Cleaning up unreferenced files.");
```

Comment:
Can we add more context to this log line which would help in case of debugging an issue? For example, if we want to check the distribution of these log lines per reason, will adding `reason` to log line help?

Reply:
Addressed the comments.